site/templates/collection/carousel/item.jet

```diff
@@ -17,7 +18,10 @@
       {{if showItemContent}}
         <div class="carousel-item-caption">
           {{if item.ItemType == "film"}}
-            {{yield sponsor(hideLink="true") item.InnerItem}}
+            {{yield carouselSponsorAwardsWrapper() content}}
```

Comment:
I'm glad you know how to do this!  But... you've added a new file with 9 lines that only gets used in one place to avoid adding two lines here... seems a bit overkill!

Reply:
Re. https://github.com/shift72/core-template/pull/379#discussion_r901337363

Is there an alternative or should we just allow the wrapper unconditionally? I can only think of this as a compromise:
```
.sponsor-awards-wrapper:empty(){
  display:none;
}
```

Reply:
I was thinking more something like this in carouse/item.jet:
```
{{if len(item.InnerItem.AwardCategories) > 0 || item.InnerItem.ImageMap["Sponsor"]}}
  <div class="sponsor-awards-wrapper">
    {{yield sponsor(hideLink="true") item.InnerItem}}
    {{yield awardsCarousel() item.InnerItem}}
  </div>
{{end}}
```